src/opnsense/service/conf/actions.d/actions_filter.conf

```diff
@@ -112,3 +112,9 @@ command:/usr/local/opnsense/scripts/filter/find_table_references.py
 parameters: %s
 type:script_output
 message:check which aliases match the given IP
+
+[gateways.list]
```

Comment:
duplicate call, already in core (including script) https://github.com/opnsense/core/blob/bb35688f0ea3a88bf6a75923447eab6939a3871f/src/opnsense/service/conf/actions.d/actions_interface.conf#L91-L94

Reply:
thanks, will replace it

Reply:
They are different - I keep it for now and delete it if the other function is enough

![Screenshot_20191229_165550](https://user-images.githubusercontent.com/3713914/71559236-1c659a80-2a5c-11ea-900b-4cfcf2152c6f.png)